releasenotes/notes/fixes_10852-e197344c5f44b4f1.yaml

```diff
@@ -0,0 +1,45 @@
+---
+upgrade_providers:
+  - |
+    The :class:`.BasicSimulator` python-based simulator included in :mod:`qiskit.providers.basic_provider`
+    now includes all the standard gates (:mod:`qiskit.circuit.library .standard_gates`) up to 3 qubits. The new basis support includes:
```

Comment:
This is maybe a feature rather than an upgrade?  Fwiw, I probably wouldn't bother listing all the gates below - "includes all the standard gates in `` :mod:`qiskit.circuit.library` ``" is descriptive enough.

Reply:
done in e81740e8e26ebecd71043406f1ac33a9c7ed5d47